packages/toolpad-core/src/AppProvider/AppProvider.tsx

```diff
@@ -30,11 +48,14 @@ export type NavigationItem = NavigationPageItem | NavigationSubheaderItem | Navi
 
 export type Navigation = NavigationItem[];
 
+// TODO: hide these contexts from public API
```

Comment:
I don't think we need to export the contexts in `AppProvider/index`, if was probably just an oversight from me.

Reply:
Or we can just move the to their own file.

Reply:
We could probably also benefit from creating a context for each of the router properties as they may update at different rates